### PR TITLE
React to realod push notification

### PR DESCRIPTION
**Pull request by marvec**

### CODE REVIEW

File: src/app/core/pusher/pusher.service.ts [hunk 1]
```diff
@@ -330,7 +330,25 @@ export class PusherService implements OnDestroy {
       if (this.isCurrentWorkspace(data)) {
         this.store$.dispatch(
           new DocumentsAction.Get({
-            query: {stems: [{collectionId: data.id}], page: 0, pageSize: 100},
+            query: {stems: [{collectionId: data.id}], page: 0, pageSize: 200},
+          })
+        );
+      }
+    });
+    this.channel.bind('Collection:reload', data => {
```
Comment (kubedo8) [inline src/app/core/pusher/pusher.service.ts]: Why we send push notification with whole object? I think it's not needed in this case.

File: src/app/core/pusher/pusher.service.ts [hunk 2]
```diff
@@ -321,7 +321,7 @@ export class PusherService implements OnDestroy {
       if (this.isCurrentWorkspace(data)) {
         this.store$.dispatch(
           new DocumentsAction.Get({
-            query: {stems: [{collectionId: data.object.id}], page: 0, pageSize: 100},
+            query: {stems: [{collectionId: data.object.id}], page: 0, pageSize: 200},
```
Comment (kubedo8) [inline src/app/core/pusher/pusher.service.ts]: 200 is new magic number?

File: src/app/core/pusher/pusher.service.ts [hunk 3]
```diff
@@ -330,7 +330,25 @@ export class PusherService implements OnDestroy {
...
+          })
+        );
+      }
+    });
+    this.channel.bind('Collection:reload', data => {
+      if (this.isCurrentWorkspace(data)) {
+        this.store$.dispatch(
+          new DocumentsAction.Get({
+            query: {stems: [{collectionId: data.object.id}], page: 0, pageSize: 200},
+          })
```
Comment (kubedo8) [inline src/app/core/pusher/pusher.service.ts]: This action has filter inside for already loaded queries (look at its effect). I think we need to force refresh in this case.

File: src/app/core/pusher/pusher.service.ts (hunk 2 above)
Comment (marvec) [reply src/app/core/pusher/pusher.service.ts]: 100 was too little. It did not read the whole collection. I would even make it 500. The problem is that paging does not work properly. Remaining data are never read. WDYT?

File: src/app/core/pusher/pusher.service.ts (hunk 3 above)
Comment (marvec) [reply src/app/core/pusher/pusher.service.ts]: How do I do that?

File: src/app/core/pusher/pusher.service.ts (hunk 1 above)
Comment (marvec) [reply src/app/core/pusher/pusher.service.ts]: I used what was already there not to have to create way too much code. Do you think it is worth fixing?

File: src/app/core/pusher/pusher.service.ts (hunk 2 above)
Comment (kubedo8) [reply src/app/core/pusher/pusher.service.ts]: I think we should implement paging or read the whole collection.

File: src/app/core/pusher/pusher.service.ts (hunk 3 above)
Comment (kubedo8) [reply src/app/core/pusher/pusher.service.ts]: You can pass boolean value to action and use it in effect.

File: src/app/core/pusher/pusher.service.ts (hunk 1 above)
Comment (kubedo8) [reply src/app/core/pusher/pusher.service.ts]: Maybe it's not worth it.

**Pull request merged**